### ISSUE TITLE
[bug] LDRtoHDRCalibration: can't create curve files

### ISSUE THREAD

**Issue by johanvandyck** [alicevision/meshroom#1189]
**Describe the bug**
- I'm using Meshroom 2020.1 and put the HDR workflow before the photogrammertry pipeline:
Camerainit, PanaoromaPrepareImages, LDRtoHDRSampling, LDRtoHDRCalibration, LDRToHDRMerge,...
I'd like to test the HDR merge functions.
- The LDRtoDHRCalibration step is always failing with **"can't create tune Curve".**

**To Reproduce**
- Input: one set of 5 *.ARW files from my Sony RX100-M7. (RAW images with sRGB colorrange; 2EV range in 5 pictures) to test the flow. The first steps (Camerainit, PanaoromaPrepareImages, LDRtoHDRSampling) are working fine.
- dropbox link to input files (85Mb): [https://www.dropbox.com/s/s0gb2wb6fmekrre/testmr-hdr.zip?dl=0](url)
- note: my camera is not yet auto-recognised. I have a pull request open. I updated my local cameradb manually; Focal length = 13.2

-  output LDRtoHDRCalibration:
```
Program called with the following parameters:
 * calibrationMethod =  Unknown Type "enum ECalibrationMethod"
 * calibrationWeight = "gaussian"
 * channelQuantizationPower = 10
 * input = "C:/Users/x/Pictures/meshroom-test/MeshroomCache/PanoramaPrepareImages/912c0205b54cb6b27e207163ad629f96340bdb1c/cameraInit.sfm"
 * maxTotalPoints = 1000000
 * nbBrackets = 5
 * response = "C:/Users/x/Pictures/meshroom-test/MeshroomCache/LdrToHdrCalibration/d21b3f0f6a76f77a316dbab29c968881df5769a8/response.csv"
 * samplesFolders = "C:/Users/x/Pictures/meshroom-test/MeshroomCache/LdrToHdrSampling/4f38b3e25a385c82124b637b4c56433e62b5f87d"
 * verboseLevel = "trace"

[19:49:32.853228][trace] 0 extrinsics declared and 5 extrinsics used.
[19:49:32.853228][info] Number of brackets automatically detected: 5. It will generate 1 hdr images.
[19:49:32.853228][info] Analyzing samples for each group
[19:49:43.286574][info] Extracting samples for each group
[19:49:46.499640][info] Start calibration
[19:49:46.499640][info] Debevec calibration with 718904 samples.
[19:58:49.492460][fatal] Can't create curves file

```

**I don't see a more elaborate error. How can I debug further?**
I have the same behaviour playing around with some attributes of the block.
During calculation, RAM is filling up till max available. (but virtual memory is not depleted)

**Desktop (please complete the following and other pertinent information):**
 - OS: win10, latest
 - Meshroom version: 
   - Binary version 2020.1 standard release version
 - hardware: intel I5-4570, 24Gb ram
 - Gforce GTX 760 (not used for this calculations)

**Additional context**
All ideas welkom to debug further.

Thanks.

**Comment by natowi** [comment]
Works for me:
![hdrm](https://user-images.githubusercontent.com/8138537/102724388-8c290100-430f-11eb-93b0-cb4a0f4d5c14.JPG)

You need to avoid camera movement when capturing the hdr image series

**Comment by johanvandyck** [comment]
Hi

It is working, but... speed?
- I tried to create a HDR image in Lightroom: only 2/5 taken into account. Indeed to much motion in the source files. Thanks Natowi.
- I took new pictures with tripod. I checked them in Lr: ok. Check in MeshRoom: not yet ok -> AVG.
- AVG: I created my meshroom project in /.../pictures/project.mg. Also the cache is written there. The LDrToHdrCalibration tries to write a response.csv file. -> AVG does not like a *.csv file in the picture (sub)folder. I added an exception.

--> everything is working fine. However one question remains: speed + why?
- There is nearly no information about the different HDR steps in the meshroom wiki. Also google or the internet has not yet discoverd the HDR process.
- Speed: Lightroom is much faster in creating my HDR images. Is there any reason why I should choose Meshroom? (I leave all parameters default; I noticed the sRGB colorspace was used (not adobe colorspace, also present in my RAW files)

Kind regards

Johan.
